pandas/io/parsers.py

```diff
@@ -120,7 +120,7 @@
     data rather than the first line of the file.
 names : array-like, optional
     List of column names to use. If file contains no header row, then you
-    should explicitly pass ``header=None``. Duplicates in this list are not
```

Comment:
‘If the file contain a header row, then you should explicitly pass ‘’header=0’’ to override the column names.’ - I think that would make more sense?

Reply:
Thank you. I've rechecked who the system is working (I've added the comment with the details) and I have updated the text as you have suggested.